recipes-tag/mpich/meta.yaml

```diff
@@ -23,6 +23,7 @@ requirements:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
         - {{ compiler('fortran') }}
+        - conda-build >=3.17.0
```

Comment:
Good point! I tried to rebuild the Docker image for `nsls2/debian-with-miniconda` with the newer version of conda-build, but it failed on some strange error. Will retry it later today. Anyway, your addition will fix it for now.

Reply:
No problem. Is the docker image new as of now? I just tried building this commit with it and it works fine.

Reply:
No, it wasn't updated to have `conda-build >=3.17.0`, however it's a good practice to have this dependency explicit. I'll try to rebuild our image so that we have the updated version of conda-build out-of-the-box.